src/stories/plans/stepsTable.stories.mdx

```diff
@@ -16,41 +16,58 @@ used to display information about each **Step** involved in installing a
 If a pre-install validation check is required, the **StepsTable** initially only
 displays **Step** names and types.
 
-<Canvas>
-  <Story id="plans-stepstable-examples--pre-validation" />
-</Canvas>
+<Story
+  id="plans-stepstable-examples--plan-steps&args=preflight:None"
+  height="320px"
+/>
 
 Once the pre-install validation is complete, an `Install` column appears. If a
 **Step** is required, the `Install` column for that **Step** features a
-checkmark. If it is optional, a selectable checkbox is shown instead.
+checkmark. A pre-validation validation can also mark a **Step** as optional. If
+it is optional, a selectable checkbox is shown instead.
+
+<Story
+  id="plans-stepstable-examples--plan-steps&args=preflight:Success"
+  height="320px"
```

Comment:
@dvdherron For some reason, when we embed a story with `args` like this, it renders in an `iframe` -- which is okay, but means we need to manually set the `height`.